package ru.practicum.shareit.util.exception;

public class FoundException extends RuntimeException {
    public FoundException(String message) {
        super(message);
    }
}
